client/core.go

```diff
@@ -9,13 +9,6 @@ import (
 	"time"
 )
 
-// Entry statuses.
-const (
-	EntryStatusUnread  = "unread"
-	EntryStatusRead    = "read"
-	EntryStatusRemoved = "removed"
-)
```

Comment:
The package `client` is the Golang API client. This code is not shared with the application. People can import this package into their own applications. As such, these constants should not be changed.